helm/default-apps-cloud-director/templates/apps.yaml

```diff
@@ -52,9 +52,9 @@ spec:
 {{- end }}
 {{- if .extraConfigs }}
   extraConfigs:
-  {{- range .extraConfigs }}
-  - kind: {{ .kind }}
-    name: {{ .name }}
+  {{- range $extraConfig := .extraConfigs }}
```

Comment:
@tuladhar Does this work when `.name` is just a string too?

Reply:
Yes, it does.
```
 (teleport)> yq '.apps.teleportKubeAgent.extraConfigs' helm/default-apps-cloud-director/values.yaml
- kind: configMap
  name: "bigmac-teleport-kube-agent-config"
```
```
 (teleport)> helm template helm/default-apps-cloud-director/ -f helm/default-apps-cloud-director/ci/ci-values.yaml | grep extraConfigs -A3
  extraConfigs:
  - kind: configMap
    name: bigmac-teleport-kube-agent-config
    namespace: default
```

```
 (teleport)> cat helm/default-apps-cloud-director/templates/apps.yaml | grep "range \$extraConfig" -A5
  {{- range $extraConfig := .extraConfigs }}
  - kind: {{ $extraConfig.kind }}
    name: {{ tpl $extraConfig.name $ }}
    namespace: {{ $.Release.Namespace }}
  {{- end }}
 {{- end }}
```

Reply:
Then it is fine.